cmd/podman/pods/create.go

```diff
@@ -73,9 +75,13 @@ func init() {
 	infraImageFlagName := "infra-image"
 	var defInfraImage string
 	if !registry.IsRemote() {
-		defInfraImage = containerConfig.Engine.InfraImage
+		if containerConfig.Engine.InfraImage != config.DefaultInfraImage {
+			defInfraImage = containerConfig.Engine.InfraImage
+		} else {
+			defInfraImage = fmt.Sprintf("localhost/podman-pause:%s-%d", version.Version, version.Built)
+		}
```

Comment:
I think it would be cleaner to not set a default image here at all and turn it into empty string.

Reply:
I changed it.

Reply:
```suggestion
```
I thinkt that's what Valentin suggested, setting the var above and killing the else

Reply:
What I want is to not set the variable at all and leave it empty.

Reply:
In my opinion, that would not be a good idea. If the user sets infra-image via containers.conf, he would no longer see his default setting here.

Reply:
That is a good point, thanks.  Yes, in this case, please remove the else branch.

Reply:
Done!